be/Cf.Domain/Aggregates/Reviews/Review.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Cf.Domain.Aggregates.Jobs;
+
+namespace Cf.Domain.Aggregates.Reviews;
+
+public class Review : Entity
+{
+    public Guid JobId { get; set; }
+
+    public double Score { get; set; }
```

Comment:
Score does not sound correct for me, maybe `Rate` or `Rating` is more correct. Any thoughts @Ech0ke ?

Reply:
I dont agree that Rate or Rating is better but I would be open to hear other suggestions

Reply:
If `Score `was a part of service entity, then `Score `would be appropriate, but since it is in review entity, `Rating `seems like a better option